src/System.Management.Automation/commands/utility/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -713,6 +718,11 @@ public abstract class PSControl
         /// </summary>
         public bool OutOfBand { get; set; }
 
+        /// <summary>
+        /// Set explicit limit for Width. Default width is a console host width.
+        /// </summary>
+        public int? Width { get; set; }
+
```

Comment:
Current limit is [Short.MaxValue](https://github.com/dotnet/corefx/blob/ec2a6190efa743ab600317f44d757433e44e859b/src/Common/src/System/IO/PathInternal.Windows.cs#L52).  My thoughts were that we had a good reserve with `Int32`. But I agree that we can forget this forever with `uint64`.